### DISCUSSION TITLE
Render single self-contained HTML file from multiple qmd files

### DISCUSSION THREAD

**Discussion by ylaizet** [quarto-dev/quarto-cli#399] [Q&A] [answered]
Hi, I would be interested in being able to generate a self-contained HTML **single file** from a list of qmd files. The basic idea behind is to concatenate the qmd files before rendering as if it was a single file. I thought I could do it with `include-in` options but I realized that this is a raw inclusion, there is no rendering before.
This would allow a modular building of the output from different parts contained in separate files, as for the book rendering but in a single ouput file.
Is that already possible and I did not see it ; or would it be something useful for the community?

**Comment by mrazomej** [comment]


**Reply by cscheid** [reply, marked as answer]
Just to be clear, the OP wanted something that was not simply "raw inclusion", and `{{< include >}}` is very much raw inclusion.
